### ISSUE TITLE
Add support for other languages

### ISSUE THREAD

**Issue by weilah** [cakebarIO/QuickAdd-for-Todoist#2]
Right now the skill supports only english. Would be great to be able to use it in other languages as well

**Comment by cakebarIO** [comment]
Thank you for your suggestion.

I am currently working on getting the skill certified in other locales. This has proven to be more-difficult than it should be!

To start, I am trying to add this to most English-speaking countries. As for non-English-locales, I need to figure out a way to accurately translate for each.

**Comment by cakebarIO** [comment]
Just re-submitted UK skill for certification. Please see Issue https://github.com/cakebarIO/QuickAdd-for-Todoist/issues/12 for UK prerequesites.

**Comment by cakebarIO** [comment]
The UK version of the skill is live and may be tested. A minor caveat...

Due to how Alexa's natural language understanding (NLU) system interprets the phrase "to add" (as in "Alexa, tell QuickAdd to add..."), based on regional settings for the UK and its specific language model, the "to add" intent is not available for this skill in the UK at this time.

However, the following intents DO work for the UK and have been added to all locales: "to log," "to put," "to task," and "to create a task."

**Comment by thiagoeh** [comment]
Any plans to lift the regional restrictions? I live in Brazil, and the skill activation page tells me that it's not available on my region.
Is it worth opening a separate issue on github, or can it be conflated on this one?

Thanks for creating this skill, btw!

**Comment by cakebarIO** [comment]
For everyone who would like this skill in countries other than the US and UK.

I am aware of the requests...I get many requests each week via Reddit and Email...and will work on each one separately once the UK issues get figured out.

I can't release in a non-English locale without first getting English issues sorted out:

Currently, I have a double-elevated support request with Alexa Dev Support for issues why certain English words are not properly understood by Alexa in the UK when they work perfectly in the US...support has told me everything looks fine on my end and the Alexa UK simulator for the UK says it should be working also...but it is not.

As for those requesting the English version released in their non-English locale...from what I have found, Amazon does not allow this. There are, physically, not English locale release settings for countries like Germany, Spain, and Brazil...and as such, I need to translate and test each language separately...which will come in turn.

My hands are tied right now, waiting on support resolutions for the UK issues. Once those get sorted, I will be releasing in Germany next.

**Comment by Killer-Cookie** [comment]
I'm a German native speaker and I'd absolutely love to help with the adaptation for Germany! Just let me know if you need help.

**Comment by maycon** [comment]
I can help adapting to Brazilian Portuguese. :-)

**Comment by Ullulaki** [comment]
Maybe any news from the German version yet? :)

**Comment by kellerassel007** [comment]
Can't wait for German release!

**Comment by jens876** [comment]
> Can't wait for German release!

I could help to translate to german.
